config/configs/default_ad.exs

```diff
@@ -0,0 +1,7 @@
+use Mix.Config
+
+config :code_fund, CodeFundWeb.Endpoint,
+  default_ad_headline: System.get_env("DEFAULT_AD_HEADLINE"),
+  default_ad_body: System.get_env("DEFAULT_AD_BODY"),
+  default_ad_image_url: System.get_env("DEFAULT_AD_IMAGE_URL"),
```

Comment:
I'd also set these equal to empty list here instead of in the controller if the env var isn't set.